src/cli/index.ts

```diff
@@ -9,14 +9,28 @@ import cmdProject from './project';
 import cmdModule from './module';
 import cmdConfig from './config';
 import { displayBanner } from '../utils/cli-ui';
+import { warnIfOutdated } from '../utils/version-check';
+import setupAntelopeProjectLogging from '../logging';
+import { defaultConfigLogging } from '../logging';
 
 // Read version from package.json
 const packageJson = JSON.parse(readFileSync(join(__dirname, '../../package.json'), 'utf8'));
 const version = packageJson.version;
 
+setupAntelopeProjectLogging({
+  logging: defaultConfigLogging,
+  modules: {},
+  cacheFolder: '.antelope/cache',
+  envOverrides: {},
+  disabledExports: [],
+});
```

Comment:
Don't use the logging interface inside the CLI and instead use logging method from the cli utils. We will refactor that later.